.azurepipelines/Ovmf-Ubuntu-GCC5.yml

```diff
@@ -0,0 +1,144 @@
+## @file
+# Azure Pipeline build file for building a platform.
+#
+# Platform: OVMF
+# OS: Ubuntu
+# Toolchain: GCC5
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+trigger:
+- master
+- newmaster
+pr:
+- master
+- newmaster
+
+resources:
+  repositories:
+  - repository: edk2
+    type: github
+    endpoint: BobCF
+    name: BobCF/edk2
+
+jobs:
+  - job: Platform_CI
+    variables:
+      package: 'OvmfPkg'
+      vm_image: 'ubuntu-18.04'
```

Comment:
Ubuntu 18.04 is really old now. Could we use something newer like 22.04 or 24.04?